tests/unittest/llmapi/lora_test_utils.py

```
@@ -72,7 +73,20 @@ def check_phi3_lora_fused_modules_output_tp2_identical_to_tp1(
     outputs_tp2 = _generate_phi3_response_lora_fused_modules(
         llm_class, _RU_LORA_ADAPTER_PROMPTS, **extra_llm_kwargs)
 
-    assert outputs_tp1 == outputs_tp2
+    for i, (out_tp1, out_tp2) in enumerate(zip(outputs_tp1, outputs_tp2)):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)